Formula/libassuan.rb

```diff
@@ -1,10 +1,10 @@
 class Libassuan < Formula
   desc "Assuan IPC Library"
   homepage "https://www.gnupg.org/related_software/libassuan/"
-  url "https://gnupg.org/ftp/gcrypt/libassuan/libassuan-2.5.3.tar.bz2"
-  mirror "https://www.mirrorservice.org/sites/ftp.gnupg.org/gcrypt/libassuan/libassuan-2.5.3.tar.bz2"
-  sha256 "91bcb0403866b4e7c4bc1cc52ed4c364a9b5414b3994f718c70303f7f765e702"
-  license "GPL-3.0"
+  url "https://gnupg.org/ftp/gcrypt/libassuan/libassuan-2.5.4.tar.bz2"
+  mirror "https://www.mirrorservice.org/sites/ftp.gnupg.org/gcrypt/libassuan/libassuan-2.5.4.tar.bz2"
+  sha256 "c080ee96b3bd519edd696cfcebdecf19a3952189178db9887be713ccbcb5fbf0"
+  license "GPL-3.0-only"
```

Comment:
the license seems little bit complicated, let's file a separate on this.

This is what I see:

```
/* assuan.c - Global interface (not specific to context).
 * Copyright (C) 2009 Free Software Foundation, Inc.
 * Copyright (C) 2001, 2002, 2012, 2013 g10 Code GmbH
 *
 * This file is part of Assuan.
 *
 * Assuan is free software; you can redistribute it and/or modify it
 * under the terms of the GNU Lesser General Public License as
 * published by the Free Software Foundation; either version 2.1 of
 * the License, or (at your option) any later version.
 *
 * Assuan is distributed in the hope that it will be useful, but
 * WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 * Lesser General Public License for more details.
 *
 * You should have received a copy of the GNU Lesser General Public
 * License along with this program; if not, see <http://www.gnu.org/licenses/>.
 * SPDX-License-Identifier: LGPL-2.1+
 */
```

Reply:
```
                              Libassuan
                             ===========

    Copyright (C) 2001-2013 Free Software Foundation, Inc.
    Copyright (C) 2001-2020 g10 Code GmbH


    This is a general purpose IPC library which is for example used
    GnuPG, GPGME and some other software.

    See COPYING.LIB on how to share, modify and distribute the
    software itself (LGPLv2.1+) and COPYING for the documentation
    (GPLv3+).
```

Reply:
Strange -- the `COPYING` file inside `libassuan-2.5.4.tar.bz2` is just the standard GPLV3 file:
```
		    GNU GENERAL PUBLIC LICENSE
		       Version 3, 29 June 2007

 Copyright (C) 2007 Free Software Foundation, Inc. <http://fsf.org/>
 Everyone is permitted to copy and distribute verbatim copies
 of this license document, but changing it is not allowed.
```

Reply:
`LGPL-2.1-or-later`?

Reply:
the license file with header declarations are the same for LGPL only and LGPL or later. So let's file a separate license PR :)